command/loadbot/generator/erc_generators.go

```diff
@@ -0,0 +1,62 @@
+package generator
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/crypto"
+)
+
+// call ERC20 contract method and encode parameters
+func NewERC20Generator(params *GeneratorParams) (*ContractTxnsGenerator, error) {
+	gen := &ContractTxnsGenerator{}
+
+	gen.BaseGenerator = BaseGenerator{
+		failedTxns: make([]*FailedTxnInfo, 0),
+		params:     params,
+		signer:     crypto.NewEIP155Signer(params.ChainID),
+	}
+
+	buf, err := hex.DecodeString(params.ContractArtifact.Bytecode)
+	if err != nil {
+		return nil, fmt.Errorf("unable to decode bytecode, %w", err)
+	}
+
+	gen.contractBytecode = buf
+	gen.contractBytecode = append(gen.contractBytecode, params.ConstructorArgs...)
+
+	if gen.encodedParams, err = params.ContractArtifact.ABI.Methods["transfer"].Encode(
+		[]string{params.RecieverAddress.String(),
+			"3",
+		}); err != nil {
+		return nil, fmt.Errorf("cannot encode ERC20 transfer method params: %w", err)
+	}
+
+	return gen, nil
+}
+
+// call ERC721 contract method and encode parameters
+func NewERC721Generator(params *GeneratorParams) (*ContractTxnsGenerator, error) {
+	gen := &ContractTxnsGenerator{}
+
+	gen.BaseGenerator = BaseGenerator{
+		failedTxns: make([]*FailedTxnInfo, 0),
+		params:     params,
+		signer:     crypto.NewEIP155Signer(params.ChainID),
+	}
+
+	buf, err := hex.DecodeString(params.ContractArtifact.Bytecode)
+	if err != nil {
+		return nil, fmt.Errorf("unable to decode bytecode, %w", err)
+	}
+
+	gen.contractBytecode = buf
+	gen.contractBytecode = append(gen.contractBytecode, params.ConstructorArgs...)
+
+	if gen.encodedParams, err = params.ContractArtifact.ABI.Methods["createNFT"].Encode(
```

Comment:
btw, what is `createNFT` method? It's not included in ERC721 standard. Loadbot for ERC721 is checking performance on minting?

Reply:
I was using teh OpenZeppelin framework and the createNFT is just a custom method that mints tokens.
The whole thing is very simple:
```
// SPDX-License-Identifier: GPL-3.0
pragma solidity >=0.7.0 <0.9.0;

import "@openzeppelin/contracts/utils/Counters.sol";
import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";


contract ZexNFTs is ERC721URIStorage {

    using Counters for Counters.Counter;
    Counters.Counter private _tokenIds;
    
    //constructor for an ERC721 is a name and symbol
    constructor (string memory tokenName, string memory tokenSymbol ) ERC721(tokenName, tokenSymbol){}

    //after we mint the token we are going to return the id of the token
    function createNFT(string memory tokenURI) public returns (uint256) {

    //get number from token counter
        uint256 newNFTTokenId = _tokenIds.current();

    //safely mint token for the person that called the function
        _safeMint(msg.sender, newNFTTokenId);
    
    //set the token uri of the token id of the uri passed
        _setTokenURI(newNFTTokenId, tokenURI);
    
    //increment the counter
        _tokenIds.increment();
        
    //return the token id
        return newNFTTokenId;
    }

}
```

Should we maybe post this in our pub docs ?

Reply:
To be honest, it's not simulating NFT transfer, am I right? I thought that was problem.

Reply:
Yes you're right. This loadbot erc721 mode is only minting the tokens, its not doing any transfers. That was the test for erc721 mode, that we've agreed upon and it's what was stated in [this task](https://linear.app/trapesys/issue/EDGE-390/erc-721-performance-tests). We can easily fix this or add an additional mode to stress test the actual erc721 token transfers in the future, if needed ...

Reply:
@ZeljkoBenovic OK. I checked the task. You're right. Resolving